### DISCUSSION TITLE
Current status of React Server Components support

### DISCUSSION THREAD

**Discussion by KlotzJesse** [prisma/prisma#10770] [Q&A]
Hello,

since january there was / is "react-prisma" supporting RSC, without full support, more experimental.

- What is the current status on it?
- Will it support also client side components?
- Could the old demo be updated to an recommended template?
- Any estimates?

Since it's in RC and shortly before being released, it would be awesome to use it in next.js 12

Thanks.
Best Regards

**Comment by nikolasburk** [comment]
Hey Jesse, thanks for raising these questions!

> What is the current status on it?

We haven't done much work on the package since the initial release and it's currently not a priority for our Product teams unfortunately. However, there actually is only [very little source code for this package](https://github.com/prisma/prisma/blob/main/packages/react-prisma/src/index.ts) (~120 LOC), so I'm pretty sure that we'd be able to merge any community PRs that keep the package up to date.

> Will it support also client side components?

This is certainly an interesting use case but I think it's one that's also not really a priority for us. Can you describe your scenario for this use case a bit more maybe? Ideally, you can even [open a new feature request](https://github.com/prisma/prisma/issues/new/choose) so that our Product and Engineering teams can evaluate this.

> Could the old demo be updated to an recommended template?

What do you mean exactly with a "recommended template"? We'll try to keep the demo in a working state at least, but this also depends on the maintenance of the `react-prisma` package I guess...

> Any estimates?

Since it's not a priority and our Engineering teams are currently not planning to pick up the work on this, there are no estimates unfortunately. I will try to advocate for this internally as well because I think it's an important thing for us to work on, but it would be great if you could [create a feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) in a GitHub issue since that's how the Product and Engineering teams are filling their backlog (GitHub Discussions are only for questions but not feature requests 🙂).

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
